devito/types/dense.py

```diff
@@ -102,6 +102,8 @@ def __eq__(self, other):
         # is that they are indeed the same exact object
         return self is other
 
+    _subs = Differentiable._subs
```

Comment:
this line should appear below all special methods lines ie those starting with `__` such as `__hash__`

Reply:
I prefer it here with `__eq__` that is sympy overwrite first then our stuff.